cmake/rigel.cmake

```diff
@@ -26,3 +26,22 @@ function(rigel_define_wasm_targets_for_dependencies)
         "SHELL:-s USE_BOOST_HEADERS=1"
     )
 endfunction()
+
+
+function(rigel_disable_warnings target)
+    if (MSVC)
+        target_compile_options(${target} PRIVATE
+            /w
```

Comment:
MSVC supports both `/` and `-` for flags, so you could use `-w` with all compilers.

Reply:
Sounds good